xonsh/pyghooks.py

```diff
@@ -281,7 +282,8 @@ def __init__(self, style_name='default'):
         style_name : str, optional
             The style name to initialize with.
         """
-        self.trap = {}  # for custom colors
+        self.trap = {} # for traping custom colors set by user
+        self._mod = {} # for internal modification of the style
```

Comment:
No. But I need to reset it every time the style changes or the env-var is toggled. If the user (e.g. in xonshrc file) or others have added something to the style that would be cleared. That would be kind of rude. I would mind removing it again if it is OK to clear the trap dict.